firestore.rules

```diff
@@ -4,7 +4,11 @@ service cloud.firestore {
     // participantID must be in the registered_participants array in the registered_studies/{studyID} document
     match /participant_responses/{studyID}/participants/{participantID} {
       allow create, read: 
-      if participantID in get(/databases/$(database)/documents/registered_studies/$(studyID)).data.registered_participants;
+      // Allows any combination of studyID and participantID to be created in Firebase
+      if true
+      // Forces participantID and studyID to already be in Firebase to log in
```

Comment:
Is there a way for researchers to preregister the participantID and studyID into Firebase? Perhaps add a link to the instructions here? And update documentation?

Also, part of me kind of feels like there should be a stricter default than a looser one. And then the documentation should make it clearer. However, from the way this PR is framed it sounds like like most researchers have pivoted to prolific and this is somewhat archaic

Reply:
Yeah, the idea of pre-registering participants isn't really a thing for online studies. It's mostly a hold out from clinic based studies where the participants are registered way ahead of time. 

I'll add a comment for a discussion post on this though!